examples/app-pg/main.py

```diff
@@ -3,25 +3,50 @@
 import os
 import time
 import pyodbc
+from faker import Faker
+from dotenv import load_dotenv
 
-CONNECTION_STRING = 'DRIVER={{PostgreSQL Unicode}};SERVER={server};DATABASE={database};UID={username};PWD={password};'
+load_dotenv()
+
+fake = Faker()
 
+CONNECTION_STRING = 'DRIVER={{PostgreSQL Unicode}};SERVER={server};DATABASE={database};UID={username};PWD={password};'
 
 SQL_CREATE_TABLE = '''
-    CREATE TABLE fruits (
+    CREATE TABLE users (
         id INT,
         name VARCHAR(50),
-        quantity INT
+        city VARCHAR(50)
     )
 '''
 
-SQL_INSERT_DATA = 'INSERT INTO fruits (id, name, quantity) VALUES (?, ?, ?)'
+SQL_INSERT_DATA = 'INSERT INTO users (id, name, city) VALUES (?, ?, ?)'
+
+def set_env():
+    os.environ[DATA_COUNT] = sys.argv[1]
+
+
+data = []
+default_count = int(os.getenv('DATA_COUNT'))
```

Comment:
Move these on the top. Also we don't need to get it from env var, we just wanted a simple example.

Define a constant value.
```
RECORD_COUNT = 10000
```

Reply:
@kabirbaidhya dai I am a bit confused, do we generate 10000 data by default or do we let users choose how much data they want to print out?

Reply:
@yankeexe Don't need to allow users to generate data. Just create a constant `RECORD_COUNT` is what @kabirbaidhya is trying to say.